packages/chain-adapters/src/evm/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -466,7 +466,7 @@ describe('EthereumChainAdapter', () => {
       } as unknown as BuildSendTxInput<KnownChainIds.EthereumMainnet>
 
       await expect(adapter.buildSendTransaction(tx)).rejects.toThrow(
-        'EthereumChainAdapter: to is required',
+        'Ethereum ChainAdapter: to is required',
```

Comment:
Use the `ChainAdapterName` enum.

Reply:
f98799710837fde0a943a3e4e0e0259650dbb8f0